### ISSUE TITLE
How we can customise the order of test suits execution

### ISSUE THREAD

**Issue by khokhariya30oct** [sorry-cypress/sorry-cypress#504]
- [ ] I have searched issues for similar feature requests

## What is this feature about (1 sentence)
Well, basically this feature is about test suits that are dependent on one other.
## Why is it needed? What is the value? For whom do we build it?
Sometimes it is required to make dependency on other test suit execution. Like executing test suit 1 store some data to the `cy.task`
and once it finishes the execution, test suit 2 should be executed by getting the stored value.
So, If we can **customize the order of execution**, we can achieve it.

## High-level feature overview
Basically, this feature is like to run test suit in ordered manner rather than only random test suit pick up. If there is order defined in `cypress.json` file then it can be given first priority.

**Comment by agoldis** [comment]
Unfortunately it's not supported in cypress, I would recommend to merge the dependent tests in a single test suite.
